### PR TITLE
Creating a user also creates an organization.

### PR DESCRIPTION
**Pull request by kurtwheeler**
## Issue Number

#76 

## Purpose/Implementation Notes

Every user is gonna need an organization! This creates them in a transaction in the create user route so we can never have a user without one.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

I created a user with a curl request and it created an org for her!

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: api/resources_portal/test/test_views.py [hunk 1]
```diff
@@ -34,6 +34,8 @@ def test_post_request_with_valid_data_succeeds(self):
         eq_(user.username, self.user_data.get("username"))
         ok_(check_password(self.user_data.get("password"), user.password))
 
+        ok_(Organization.objects.get(owner=user))
```
Comment (arielsvn) [inline api/resources_portal/test/test_views.py]: Are we using `nose` helpers? Instead of `self.assert...`?

Comment (kurtwheeler) [reply api/resources_portal/test/test_views.py]: Hey man, you wrote the rest of the test!

Comment (kurtwheeler) [reply api/resources_portal/test/test_views.py]: :P

Comment (arielsvn) [reply api/resources_portal/test/test_views.py]: hahah yes, just noticed it. It looked weird with the underscore after `ok`

Comment (kurtwheeler) [reply api/resources_portal/test/test_views.py]: But uh, that's a great question. Should we? I tend to follow the existing conventions by default but it appears you didn't set that convention on purpose.

Comment (arielsvn) [reply api/resources_portal/test/test_views.py]: yes, this file was also generated by the template, I think I wrote the permission tests with the default methods. I'd lean towards them, but we can update it in a future PR if you want.

Comment (kurtwheeler) [reply api/resources_portal/test/test_views.py]: I switched all the nose asserts over to unittest asserts.

**Pull request merged**